### PR TITLE
Minor refactoring of Sql.php

### PR DESCRIPTION
**Pull request by kamil-tekiela**
Just fixing a couple of Psalm/PHPStan issues and improving ease of reading of some code.

### CODE REVIEW

File: libraries/classes/Sql.php [hunk 1]
```diff
@@ -1336,40 +1345,41 @@ private function getHtmlForPreviousUpdateQuery(
...
+            $text = __(
+                'Current selection does not contain a unique column.'
+                . ' Grid edit, Edit, Copy and Delete features may result in'
+                . ' undesired behavior. %s'
+            );
+        } else {
+            return '';
+        }
+
+        return Message::notice(
```
Comment (williamdes) [inline libraries/classes/Sql.php:1374]: Why the else is needed ?
I am not sure to understand how it would reach the last return

Comment (kamil-tekiela) [reply libraries/classes/Sql.php:1374]: Good catch! That is clearly my mistake. I will fix it tomorrow. I might even roll it back to what it was before as my attempt to make it more readable wasn't very successful.

Comment (kamil-tekiela) [reply libraries/classes/Sql.php]: I reverted this change partially. It wasn't more readable than before.

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**